### PR TITLE
Fix Item_Base prefab inheritance and general prefab cleanup

### LINKED ISSUE #112: Actions dont work on dropped items
### What happened?

Actions dont work on dropped items.

### What did you expect to happen?

Actions should still work on dropped items.

### How can we reproduce it (as minimally and precisely as possible)?

Drop any EveronLife item (apple, tomato, etc..) that doesn't have a collider on the ground and try to pick it up via action.

This only happens if there is no other collider near the item. E.g it works fine when the item is on a table.
_*None of the items currently has a collider._

### Anything else we need to know?

**Fix**: A proper collider should be added to every item.

### EveronLife Version

<details>
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### LINKED ISSUE #150: Some prefabs do not inherit from Item_Base
### What happened?

These prefabs:
- Prefabs\Items\Drinks\WaterBottle.et
- Prefabs\Items\Food\Tomato.et
- Prefabs\Resources\Mining\* (all of them)
should all inherit from `Item_Base.et` so their own prefab data only adds additional information/changes instead of a full copy. This is probably my fault, cause I created them wrongly. Can somebody please go through all of them and fix this?

### PR DESCRIPTION
**Pull request by Arkensor**
Fix tomato, water bottle, mining bars, and ores not inheriting from item_base
Fix plum origin
Fix applejuice prefab name
Fix items overriding defaults with default values in prefabs
Fix missing collider meshes on all items

Fixes #150 
Fixes #112

### CODE REVIEW

File: src/Worlds/DebugWorld/DebugWorld_PlumChain.layer [hunk 1]
```diff
@@ -70,32 +70,32 @@ GenericEntity PlumTrader {
     Flags 1027
    }
    PlumTraderTestItems {
-    coords -0.627 0.918 0.343
```
Comment (thebonbon) [inline src/Worlds/DebugWorld/DebugWorld_PlumChain.layer:73]: Not part of this PR

File: src/Worlds/DebugWorld/DebugWorld_Terrain.layer [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
...358 358 -1 0 2972 359 -1 2 2768 -1 -1 2 1539 -1 361 0 777 -1 362 2 719 -1 363 0 338 -1 364 2 309 -1 -1 0 -17050 366 369 0 -8575 367 368 0 -4337 -1 -1 0 -12812 -1 -1 0 -25525 370 371 0 -21287 -1 -1 0 -29762 -1 -1 0 -15294 373 404 2 -17050 374 389 0 -5941 375 382 2 -8575 376 379 0 -1264 377 378 2 -4337 -1 -1 2 -4337 -1 -1 0 -1264 380 381 2 -12812 -1 -1 2 -12812 -1 -1 2 -8575 383 386 0 -10617 384 385 2 -4337 -1 -1 2 -4337 -1 -1 0 -10617 387 388 2 -12812 -1 -1 2 -12812 -1 -1 0 -5941 390 397 2 -25525 391 394 0 -1264 392 393 2 -21287 -1 -1 2 -21287 -1 -1 0 -1264 395 396 2 -29762 -1 -1 2 -29762 -1 -1 2 -25525 398 401 0 -10617 399 400 2 -21287 -1 -1 2 -21287 -1 -1 0 -10617 402 403 2 -29762 -1 -1 2 -29762 -1 -1 2 -17050 405 420 0 -24647 406 413 2 -8575 407 410 0 -19970 408 409 2 -4337 -1 -1 2 -4337 -1 -1 0 -19970 411 412 2 -12812 -1 -1 2 -12812 -1 -1 2 -8575 414 417 0 -29323 415 416 2 -4337 -1 -1 2 -4337 -1 -1 0 -29323 418 419 2 -12812 -1 -1 2 -12812 -1 -1 0 -24647 421 428 2 -25525 422 425 0 -19970 423 424 2 -21287 -1 -1 2 -21287 -1 -1 0 -19970 426 427 2 -29762 -1 -1 2 -29762 -1 -1 2 -25525 429 432 0 -29323 430 431 2 -21287 -1 -1 2 -21287 -1 -1 0 -29323 433 434 2 -29762 -1 -1 2 -29762 -1 -1
```
Comment (thebonbon) [inline src/Worlds/DebugWorld/DebugWorld_Terrain.layer:2]: Not part of this PR

File: src/Worlds/DebugWorld/DebugWorld_TomatoChain.layer [hunk 3]
```diff
@@ -70,32 +70,35 @@ GenericEntity TomatoTrader {
     Flags 1027
    }
    TomatoTraderTestItems {
-    coords -0.774 0.956 0.18
```
Comment (thebonbon) [inline src/Worlds/DebugWorld/DebugWorld_TomatoChain.layer:73]: Not part of this PR

File: src/Prefabs/Resources/Mining/IronNugget.et [hunk 4]
```diff
@@ -41,7 +40,7 @@ GenericEntity {
       ItemRenderRotation 0 0 90
      }
     }
-    m_SlotType SLOT_LOOT_STORAGE
+    m_Size SLOT_2x1
```
Comment (thebonbon) [inline src/Prefabs/Resources/Mining/IronNugget.et:43]:
```suggestion
    m_Size SLOT_1x1
```

File: src/Prefabs/Resources/Mining/GoldNugget.et [hunk 5]
```diff
@@ -41,7 +40,7 @@ GenericEntity {
       ItemRenderRotation 0 0 90
      }
     }
-    m_SlotType SLOT_LOOT_STORAGE
+    m_Size SLOT_2x1
```
Comment (thebonbon) [inline src/Prefabs/Resources/Mining/GoldNugget.et:43]:
```suggestion
    m_Size SLOT_1x1
```

File: src/Prefabs/Resources/Mining/CopperNugget.et [hunk 6]
```diff
@@ -41,7 +40,7 @@ GenericEntity {
       ItemRenderRotation 0 0 90
      }
     }
-    m_SlotType SLOT_LOOT_STORAGE
+    m_Size SLOT_2x1
```
Comment (thebonbon) [inline src/Prefabs/Resources/Mining/CopperNugget.et:43]:
```suggestion
    m_Size SLOT_1x1
```

File: src/Prefabs/Resources/Mining/GoldNugget.et (hunk 5 above)
Comment (thebonbon) [reply src/Prefabs/Resources/Mining/GoldNugget.et:43]: If 1x1 CameraDistanceToItem: 0.275

File: src/Prefabs/Resources/Mining/IronNugget.et (hunk 4 above)
Comment (thebonbon) [reply src/Prefabs/Resources/Mining/IronNugget.et:43]: If 1x1 CameraDistanceToItem: 0.275

File: src/Prefabs/Resources/Mining/CopperNugget.et (hunk 6 above)
Comment (thebonbon) [reply src/Prefabs/Resources/Mining/CopperNugget.et:43]: If 1x1 CameraDistanceToItem: 0.275

**Review by thebonbon — CHANGES_REQUESTED** [review]
Why / What did you change in the fbx and txo files?

**Comment by thebonbon** [comment]
> Remove rigid body from all items

Sand, Gravel and Cement bag still have Rigidbodies.

File: src/Worlds/DebugWorld/DebugWorld_PlumChain.layer (hunk 1 above)
Comment (Arkensor) [reply src/Worlds/DebugWorld/DebugWorld_PlumChain.layer:73]: Had to be updated because the models have a different origin now

File: src/Prefabs/Resources/Mining/IronNugget.et (hunk 4 above)
Comment (Arkensor) [reply src/Prefabs/Resources/Mining/IronNugget.et:43]: Yeah good idea to make them 1x1, I think that is better. I adjusted their rotation and scale to be this:
image

**Comment by Arkensor** [comment]
> Why / What did you change in the fbx and txo files?

I fixed the origin of those models so it is at the center bottom, so when you drag and drop them somewhere they are not halfway into the ground. I am sure there are other means but I wanted to give a good example for others so they see all our models have this 0 0 0 origin. 

> Sand, Gravel and Cement bag still have Rigidbodies.

That is correct, I only removed it from items that are so small that they will not collide with players or vehicles. Sandbags are so big I think you should be able to walk on them or have a vehicle run over. That is why I left them as is.

**Review by thebonbon — APPROVED** [review]
